R/reportEDGET.R

```diff
@@ -963,6 +934,44 @@ toolReportEDGET <- function(output_folder = ".",
     toMIF[period == 2100][, period := 2150]
   )
 
+  varsl <- list(
```

Comment:
Hi Johanna, sorry to meddle here. I am probably missing something, but to me it seems that the variables were not moved into the extended reporting but below it and would still be part of the not extended reporting. Is that right?

Reply:
Hi David- thanks for finding that stupid bug. I will test it again on a REMIND run to make sure that all duplicates are gone with the fix